isort/core.py

```diff
@@ -395,6 +402,11 @@ def process(
         for output_str in verbose_output:
             print(output_str)
 
+    if imports_only:
+        output_stream.seek(0)
```

Comment:
Will this work if output_stream is sys.stdout?

Reply:
You're right, it won't work with sys.stdout. Fixed it.